### ISSUE TITLE
[Bug Report] Cannot set id attribute on el-select input for accessibility by screen reader

### ISSUE THREAD

**Issue by pushchris** [ElemeFE/element#7728]
### Element UI version
2.0.0-rc.1

### OS/Browsers version
macOS 10.13 / Chrome 62

### Vue version
2.5.2

### Reproduction Link
[https://jsfiddle.net/mk6p076a/](https://jsfiddle.net/mk6p076a/)

### Steps to reproduce
Create any ```el-select``` element and set it's ```id``` attribute while inside of either a label or ```el-form-item```

### What is Expected?
It is expected that the ```id``` attribute will get passed through to the HTML ```input``` so that screen readers can reference the label and input item together.

### What is actually happening?
The ```id``` attribute is being attached to the upper label ```.el-select``` div
